config/data_sync_list/common.json

```diff
@@ -1,4 +1,20 @@
 {
+    "Files": [
+        {
+            "Path": "/var/log/obmc-console.log",
+            "Description": "Host console persisted log data",
+            "SyncDirection": "Active2Passive",
+            "SyncType": "Periodic",
+            "Periodicity": "PT60S"
```

Comment:
I feel interval is fine but, I would defer to data owner to share suggestion whether it need to decrease or increase.